setup.py

```diff
@@ -17,7 +17,7 @@
     license="apache-2.0",
     author="Talmaj Marinc",
     packages=PACKAGES,
-    install_requires=["torch>=1.4.0", "onnx>=1.6.0"],
+    install_requires=["torch>=1.4.0", "onnx>=1.6.0", "torchvision>=0.9.0"],
```

Comment:
Why is this requirement necessary?

Reply:
It's needed for the NonMaxSuppression operation.